starter/script.js

```diff
@@ -1 +1,54 @@
-// put code here!
+// put code here
+terrain();
+
+function terrain() {
+var terrain = prompt("Choose your terrain: (road, trail)");
+  if (terrain == "road") {
+    RoadStory();
+  }
+  else if (terrain === "trail") {
+    TrailStory();
+    }
+  }
+
+function RoadStory(){
+      var city = prompt("which city would you like? DC or NYC? (goToDC, goToNYC)");
+      if (city === "goToDC"){
+        goToDC();
+        }
+      }
+          if (city === "goToNYC"){
+        goToNYC();
```

Comment:
try to be consitent w/ code indentation. it will make your code easier to read and debug!